scripts/toggle.sh

```diff
@@ -49,22 +53,29 @@ cmd=("${@:$OPTIND}")
 
 opened="$(showvariable @__popup_opened)"
 
-if [[ -n "$opened" && ("$opened" = "$name" || -z "$*") ]]; then
+if [[ -n "$opened" ]]; then
```

Comment:
The removed check is necessary: If the specified name doesn't match the currently opened popup, we open a new popup within the current one (i.e. popup-in-popup).

Reply:
Oh I see, I was thinking something like neovim opens a popup and I also have a generically set popup in a tmux keybind, I can either use that to close the popup that is opened up by neovim or toggle a new one if that is not open.

So the neovim popup is named differently from the other one, so it was causing this popup in a popup. I just did not think of it as a feature :D

Reply:
> Oh I see, I was thinking something like neovim opens a popup and I also have a generically set popup in a tmux keybind, I can either use that to close the popup that is opened up by neovim or toggle a new one if that is not open.

I have changed the testing conditions to `[[ -z $name || $name == $opened_name ]]`, which means that an empty `--name` closes the currently opened popup. So, if you have bound a key to `run "#{@popup-toggle} bash"`, pressing that key will open Bash in your main session and will close any open popups when inside them. The user can use it as the primary toggle key.

Personally, I use several different keys to toggle various programs (fish, lazygit, yazi, etc.). I will add some notes for "popup-in-popup" later.